src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
-body {
-  margin: 0;
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+
+  &--additional {
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+    margin: 50px auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.